server/src/handlers/message_handler.rs

```diff
@@ -895,9 +895,9 @@ pub async fn create_suggested_queries_handler(
         ServerDatasetConfiguration::from_json(dataset_org_plan_sub.dataset.server_configuration);
     let base_url = dataset_config
         .LLM_BASE_URL
-        .unwrap_or("https://openrouter.ai/api/v1".into());
+        .unwrap_or("https://api.openai.com/api/v1".into());
```

Comment:
Wait are we defaulting to openrouter or open ai. The model names seem inconsistent.